javascript/NocaptchaField.js

```diff
@@ -1,12 +1,20 @@
 var _noCaptchaFields=_noCaptchaFields || [];
+var _noCaptchaValidationExemptActions=_noCaptchaValidationExemptActions || [];
 
 function noCaptchaFieldRender() {
     var submitListener=function(e) {
+        // If the action is exempt from validation, skip any recaptcha checks
+        if (e.submitter &&
+            e.submitter.name &&
+            _noCaptchaValidationExemptActions.indexOf(e.submitter.name.substring(7)) > -1) {
```

Comment:
Using substring here to remove `action_` might seem a little flaky but it's the same logic in [FormAction](https://github.com/silverstripe/silverstripe-framework/blob/4/src/Forms/FormAction.php#L128) so should be sweet